### DISCUSSION TITLE
Add .xar (yet another proprietary file type)?

### DISCUSSION THREAD

**Discussion by spikyman** [ImageMagick/ImageMagick#2998] [General]
Dear Wizards,

My image management software uses IM, which seems to be why it doesn't see .xar files. I don't need anything beyond thumbnails, and I'm guessing anything else would be very messy. 

FYI, It's output by Xara, a graphics program, whose lineage began as an open source graphics program for linux in the 80's.  Sadly, it's now owned by magix.com, and currently only for win10.  

Thanks for any consideration you might give this.

Mark

**Comment by urban-warrior** [comment]
To consider a new format you will need to post a link to the format specification, a number of example files in the format, a short list of utilities that already support the format., and an estimation of how many use the format.

**Comment by spikyman** [comment]
Thanks for getting back to me, Cristy.
I've sent your requirement to the product owners, since the last publicly
available spec is 15 years old, and they would have the rest of the data as
well. They will contact you directly if they wish to pursue the matter.
*Mark Cohen*
*434 242-1841*
